pkg/pool/operations/pool_utils.go

```diff
@@ -32,6 +32,24 @@ import (
 	"k8s.io/klog"
 )
 
+var SupportedCompressionTypes = map[string]bool{
```

Comment:
nit: This kind of validation should be incorporated with openAPISchemaValidations :) 
https://github.com/openebs/zfs-localpv/pull/66 (But we will take it later).

Reply:
I am aware of this. I have not yet started to work on OpenAPI CRD schema.
Even though we will have validations in OpenAPI schema — the code will have the critical validations in place.